doc/bridge_packages/flame_lottie/flame_lottie.md

```diff
@@ -0,0 +1,58 @@
+# flame_lottie
+
+This package allows you to load and add Lottie animations to your flame game.
+
+> Lottie is a mobile library for Android and iOS that parses Adobe After Effects animations
+exported as json with Bodymovin and renders them natively on mobile!
+
+Source: [lottie-android](https://github.com/airbnb/lottie-android) on Github
+
+
+The native Lottie libraries (such as [lottie-android](https://github.com/airbnb/lottie-android))
```

Comment:
Question: does this means that desktop and web isn't supported? I wonder if there is a way to avoid native integration.

Reply:
no, quite the contrary the bridge also works on other platforms. 
To specific on platforms that are supported by the Flutter lottie package (all mobile and desktop environments as of now)

However, the **lottie** Flutter package itself, seems to be a sole re-implementation/conversion of the lottie-android libray by Airbnb, according to the package author.
> Source: https://github.com/xvrh/lottie-flutter/blob/master/README.md?plain=1#L9